### PR TITLE
don't render filters with 0 or 1 values. resolves #29

### LINKED ISSUE #29: unexpected settings.highlight behavior with an empty column
This line `data.forEach(function(d){d.QUERYFL=""})` plus the settings below highlights every point for some reason ... 

```
highlight:
            {value_col: 'QUERYFL'
            ,label: 'Open Query'
            ,value: 'Y'
            ,detail_col: 'QUERY'
            ,attributes:
                {'stroke': 'black'
                ,'stroke-width': '8'
                ,'fill': 'none'
                ,'stroke-opacity':"0.2"}}
```

screen shot 2017-03-06 at 3 46 41 pm

### PR DESCRIPTION
**Pull request by jwildfire**
implemented in onInit() slightly hack-y, but seems to work ok ...

### CODE REVIEW

File: src/onInit.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import lengthenRaw from './util/lengthenRaw';
 
 export default function onInit() {
+    console.log(this)
```
Comment (samussiah) [inline src/onInit.js]: stray

File: src/onInit.js [hunk 2]
```diff
@@ -38,6 +40,23 @@ export default function onInit() {
             : NaN;
     });
 
+  // Remove filters for variables with 0 or 1 levels
+  var chart = this;
+  console.log(chart)
```
Comment (samussiah) [inline src/onInit.js]: another stray

**Comment by jwildfire** [comment]
@brittsikora @emmorris Ready for Testing at: https://rhoinc.github.io/viz-library/testing/safetyExplorer-v2-0-1/ae-timeline/

Confirm that no filter for sex is being shown and a warning is being printed in the console (#29)

**Comment by brittsikora** [comment]
Confirming that it behaves as expected in Firefox, IE, and Chrome

**Pull request merged**